src/pages/EntityDetails/counts.js

```diff
@@ -7,13 +7,24 @@ export const incrementCounts = (status, counts) => {
   return counts;
 };
 
-export const generateSecondaryCounts = (modelStatusData, segment, selector) =>
+const generateOfferCounts = (modelStatusData) => {
+  let offerCount = 0;
+  Object.entries(modelStatusData["offers"]).forEach((offer) => {
+    if (offer[1]["total-connected-count"] > 0) {
+      offerCount++;
```

Comment:
So if there were 3 connections you only want the count to go up by 1?

Reply:
Good catch!